prepare-image.sh

```diff
@@ -11,6 +11,7 @@ if [[ ! -z ${EXTRA_PKGS_LIST:-} ]]; then
         xargs -rtd'\n' dnf --setopt=install_weak_deps=False install -y < /tmp/${EXTRA_PKGS_LIST}
     fi
 fi
+dnf install -y https://download-ib01.fedoraproject.org/pub/epel/8/Everything/x86_64/Packages/i/inotify-tools-3.14-19.el8.x86_64.rpm
```

Comment:
This will break once this version is deleted. What you need is to install epel-release but disable the added repo via dnf (`dnf config-manager --set-disabled epel` or something like that). Then you can use `--enablerepo=epel` when installing inotify-tools specifically.